### ISSUE TITLE
Cannot validate XML character data with emoji.

### ISSUE THREAD

**Issue by tmathews** [michaelkourlas/node-xmlcreate#3]
Try this on for size:

```
function validateChar(str) {
    var charRegex = "\\u0009|\\u000A|\\u000D|[\\u0020-\\uD7FF]";
    var surrogateCharRegex = "[\\uD800-\\uDBFF][\\uDC00-\\uDFFF]";
    return new RegExp("^((" + charRegex + ")|(" + surrogateCharRegex + "))*$")
        .test(str);
}
console.log(validateChar('❤️'));
console.log(validateChar('😃'));
```

This pretty much breaks any other module that uses this.

**Comment by michaelkourlas** [comment]
Yep, can't believe I missed a whole block. Submitting a fix now.

**Comment by michaelkourlas** [comment]
This is now fixed in 1.0.2.
